lib/plugins/aws/provider.js

```diff
@@ -454,7 +454,7 @@ class AwsProvider {
           },
           awsLambdaTimeout: { type: 'integer', minimum: 1, maximum: 900 },
           awsLambdaTracing: { anyOf: [{ enum: ['Active', 'PassThrough'] }, { type: 'boolean' }] },
-          awsLambdaVersionning: { type: 'boolean' },
+          awsLambdaVersioning: { type: 'boolean' },
```

Comment:
I believe it's safe to change given it's a definition name

Reply:
correct :+1: